preview-src/index.css

```diff
@@ -82,13 +82,18 @@ body .comment-container .avatar-container a {
 	display: flex;
 }
 
-body .comment-container .avatar-container img.avatar {
+body .comment-container .avatar-container img.avatar,
+body .comment-container .avatar-container .avatar-icon svg {
 	margin-right: 0;
 }
 
-body img.avatar {
+body img.avatar,
+body span.avatar-icon svg {
```

Comment:
Did we decide to use a conditional `filter: invert(100%);` rule here, to get the default avatar icon to align with the theme?